test/e2e/util.go

```diff
@@ -420,29 +420,29 @@ func TearDown(namespace string) {
 		return client.DeleteAllOf(clientCtx, &operatorsv1alpha1.InstallPlan{}, inNamespace)
 	}).Should(Succeed(), "failed to delete test installplans")
 
-	Eventually(func() (remaining []operatorsv1alpha1.InstallPlan, err error) {
-		list := &operatorsv1alpha1.InstallPlanList{}
-		err = client.List(clientCtx, list, inNamespace)
-		if list != nil {
-			remaining = list.Items
+	var installPlangvr = schema.GroupVersionResource{Group: "operators.coreos.com", Version: "v1alpha1", Resource: "installplans"}
+	Eventually(func() ([]unstructured.Unstructured, error) {
+	fmt.Printf("checking installplan\n")
```

Comment:
Is there a reason you chose to use the `fmt` package here?
```suggestion
	logf("checking installplan\n")
```

Reply:
Sorry, it was for my debug.  I'll take out.